### ISSUE TITLE
Program resources not loaded when installed in directory with special characters

### ISSUE THREAD

**Issue by wzging** [Beep6581/RawTherapee#7072]
I found a bug about rt5.10 when i installed it with chinese path (eg "D:/soft/中文/RawTherapee") in Windows。
<img width="1073" alt="企业微信截图_97cfe3f8-7fe5-4588-a4b2-5582ec2a114e" src="https://github.com/Beep6581/RawTherapee/assets/167062683/9b60e60d-f86d-4225-b8b2-1a12fdd9dd74">
<img width="1105" alt="企业微信截图_191f063d-40a6-4023-a130-9d47534ed7c7" src="https://github.com/Beep6581/RawTherapee/assets/167062683/ab806d73-e632-4ea7-974b-2fc747395d98">
when i open the app to load cr3 file  and select TP_ICM_INPUTCAMERA the picture became dark than select TP_ICM_INPUTCAMERAICC

**Comment by Lawrence37** [comment]
Camera standard looks more right to me. The DCP is over-saturated for some reason.

Not relevant to this issue, but what language do you have selected? It's weird that you are seeing "TP_ICM_INPUTCAMERA" and similar instead of readable words like "Camera standard" (English).

**Comment by wzging** [comment]
> Camera standard looks more right to me. The DCP is over-saturated for some reason.
> 
> Not relevant to this issue, but what language do you have selected? It's weird that you are seeing "TP_ICM_INPUTCAMERA" and similar instead of readable words like "Camera standard" (English).

i install RT select engilsh language,but the installed path contains chinese like this"D:/soft/中文/RawTherapee",and i also try to install in english path to load cr3 file became normal。 “TP_ICM_INPUTCAMERA”is a selection on the ui panel. When i selected “TP_ICM_INPUTCAMERA” option there is a config:InputProfile=(camera)  in pp3 file. If i selected “TP_ICM_INPUTCAMERAICC” option there is a config: InputProfile=(cameraICC) in pp3 file. 
<img width="527" alt="image" src="https://github.com/Beep6581/RawTherapee/assets/167062683/5fcf7668-fa84-4207-90d1-9e9934e005f9">
<img width="397" alt="image" src="https://github.com/Beep6581/RawTherapee/assets/167062683/cb9267a7-50df-4509-a001-55dba6fd4477">

**Comment by Lawrence37** [comment]
What I mean is the text you are seeing in the UI is not supposed to be like that. When English is selected, it should look like this:
![image](https://github.com/Beep6581/RawTherapee/assets/45837045/50f52be7-0fc7-4f35-9195-e609d4d61d22)
And if you select Chinese:
![image](https://github.com/Beep6581/RawTherapee/assets/45837045/2bf07528-3887-406f-a1ac-77f00594a5d7)

**Comment by wzging** [comment]
> What I mean is the text you are seeing in the UI is not supposed to be like that. When English is selected, it should look like this: ![image](https://private-user-images.githubusercontent.com/45837045/331826947-50f52be7-0fc7-4f35-9195-e609d4d61d22.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYxMjYwMTMsIm5iZiI6MTcxNjEyNTcxMywicGF0aCI6Ii80NTgzNzA0NS8zMzE4MjY5NDctNTBmNTJiZTctMGZjNy00ZjM1LTkxOTUtZTYwOWQ0ZDYxZDIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE5VDEzMzUxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlYzZhNmIxODMzNDlhOGM5OGNkYjA1ZmE0YWY3ZDMxYTAwNmI0Y2FmY2YwZTVjMTI1ZjU0NmQxMmYxY2Q3OTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mFILZ4e2TDt06ZgYGxS4k5IKSLU1smP7l3utSMtS-vs) And if you select Chinese: ![image](https://private-user-images.githubusercontent.com/45837045/331827104-2bf07528-3887-406f-a1ac-77f00594a5d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYxMjYwMTMsIm5iZiI6MTcxNjEyNTcxMywicGF0aCI6Ii80NTgzNzA0NS8zMzE4MjcxMDQtMmJmMDc1MjgtMzg4Ny00MDZmLWExYWMtNzdmMDA1OTRhNWQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE5VDEzMzUxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM0MGYyZWUzZDE5MzRmNTljNjJjYTA5MmE3MjJmODc1NTE2MjJlNzQyZjc4YTM2NzExODIxNzBjNDhmZmI3NmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4Gzlf8JPE_31F-breOmSl24JDnGTUNE7B6F7utAzxTM)

thanks! now the problem is when i use the pp3 file contains "InputProfile=(camera)"in the part of Color Management  like this pic to load cr3 raw files will become dark(RawTherapee installed in "D:/soft/中文/RawTherapee"),but when i install RT in"D:/soft/RawTherapee" which do not contain chinese words it will work well and do not become dark.I tried to compile the RT project to solve this problem, but I don't know which part of the code will trigger the InputProfile problem. Can you give me some tips about the InputProfile implementation ?
![image](https://github.com/Beep6581/RawTherapee/assets/167062683/d11d25e4-f6de-4952-9265-65f7a3cdad1b)

**Comment by Lawrence37** [comment]
The Camera standard option uses the color matrix in camconst.json. Perhaps it is unable to find the file due to the Chinese characters in the path? When you install it to D:/soft/RawTherapee, does the UI also show "TP_ICM_INPUTCAMERA", or does it show "Camera standard"?

**Comment by wzging** [comment]
> The Camera standard option uses the color matrix in camconst.json. Perhaps it is unable to find the file due to the Chinese characters in the path? When you install it to D:/soft/RawTherapee, does the UI also show "TP_ICM_INPUTCAMERA", or does it show "Camera standard"?

yeah, when i install it to "D:/soft/RawTherapee",the UI show like this 
<img width="498" alt="image" src="https://github.com/Beep6581/RawTherapee/assets/167062683/ec288a7e-bb96-432c-afb8-e40ac71aa219">
The problem (become dark)I encountered （RT installs in "D:/soft/中文/RawTherapee",and set InputProfile=(camera)）only occurs in cr3 format, and has not been found in other raw formats.

**Comment by Lawrence37** [comment]
Ok, I think the two issues are the same. RawTherapee is unable to load some program files when installed in that location.

Not all cameras rely on camconst.json. That's why you are seeing only some cameras are affected.
